ts/features/payments/bizEventsTransaction/screens/PaymentsTransactionBizEventsCartItemDetailsScreen.tsx

```diff
@@ -0,0 +1,101 @@
+import * as React from "react";
+import { ScrollView } from "react-native";
+import {
+  Divider,
+  H6,
+  IOStyles,
+  ListItemInfo
+} from "@pagopa/io-app-design-system";
+import { RouteProp, useRoute } from "@react-navigation/native";
+import { PaymentsTransactionBizEventsParamsList } from "../navigation/params";
+import I18n from "../../../../i18n";
+import { CartItem } from "../../../../../definitions/pagopa/biz-events/CartItem";
+import { UserDetail } from "../../../../../definitions/pagopa/biz-events/UserDetail";
+import { formatAmountText } from "../utils";
+import { IOScrollViewWithLargeHeader } from "../../../../components/ui/IOScrollViewWithLargeHeader";
+
+export type PaymentsTransactionBizEventsCartItemDetailsScreenParams = {
+  cartItem: CartItem;
+};
+
+export type PaymentsTransactionBizEventsCartItemDetailsScreenProps = RouteProp<
+  PaymentsTransactionBizEventsParamsList,
+  "PAYMENT_TRANSACTION_BIZ_EVENTS_CART_ITEM_DETAILS"
+>;
+
+const PaymentsTransactionBizEventsCartItemDetailsScreen = () => {
+  const route =
+    useRoute<PaymentsTransactionBizEventsCartItemDetailsScreenProps>();
+  const { cartItem } = route.params;
+
+  const getDebtorText = (debtor: UserDetail) => {
+    const debtorNameLabel = debtor.name ? <H6>{debtor.name}</H6> : <></>;
+    const debtorCodeLabel = debtor.taxCode ? (
+      <H6>({debtor.taxCode})</H6>
+    ) : (
+      <></>
+    );
+    return (
+      <>
+        {debtorNameLabel}
+        {debtorCodeLabel}
+      </>
+    );
+  };
```

Comment:
Prefer `null` when you want to render **nothing**. Also, we could simplify as follows:
```suggestion
const getDebtorText = (debtor: UserDetail) => {
  return (
    <>
      {debtor.name ? <H6>{debtor.name}</H6> : null}
      {debtor.taxCode ? <H6>({debtor.taxCode})</H6> : null}
    </>
  );
};
```
What do you think?

Reply:
And what happens if they are both `undefined`? Is it okay to display a list item info with an empty value?

Reply:
What do you think on what implemented in this commit? a6ea86a715af8f4339ccbf7a80d8a2f14324d929